src/doc/rustc/src/platform-support/loongarch-linux.md

```diff
@@ -0,0 +1,86 @@
+# loongarch\*-unknown-linux-\*
+
+**Tier: 3**
+
+[LoongArch] is a new RISC ISA developed by Loongson Technology, based in China.
+
+[LoongArch]: https://loongson.github.io/LoongArch-Documentation/README-EN.html
+
+The target name follow this format: `<machine>-<vendor>-<os><fabi_suffix>, where `<machine>` specifies the CPU family/model, `<vendor>` specifies the vendor and `<os>` the operating system name.
```

Comment:
Self note: rewrite this document in 2 days.

Reply:
Hmm. I'll probably have to hurry a bit...

Reply:
Should we try ChatGPT here? :)

Reply:
I'll be revising this doc later in a separate PR (at least one change to the tier is definitely going to happen), so consider my previous comment to no longer block this PR.